modules/cnv_configuring_masquerade.adoc

```diff
@@ -0,0 +1,35 @@
+[[configuring-masquerade]]
+=== Configuring masquerade mode
+
+You can use masquerade mode to hide a virtual machine's outgoing traffic behind
+the pod IP address. Masquerade mode uses Network Address Translation (NAT) to
+connect virtual machines to the pod network backend through a Linux bridge.
+
+Enable masquerade mode and allow traffic to enter the virtual machine by
+editing your virtual machine configuration file.
+
+.Prerequisites
+
+* The virtual machine must be configured to use DHCP to acquire IPv4 addresses.
+
+.Procedure
+
+. Edit the `interfaces` spec of your virtual machine configuration file:
++
+[source,yaml]
+----
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          masquerade: {} <1>
+          ports:
+            - port: 80 <2>
+  networks:
+  - name: red
+    pod: {}
+----
+<1> Connect using masquerade mode
```

Comment:
I think using the word 'connect' is misleading, I prefer interface type here.

Reply:
interface type is something different: virito, e1000, etc. so I wouldn't use it here.

We sometime use "bind", but I don't really know why @myakove find "connect" misleading.

Reply:
@myakove The "Connect using masquerade mode" was taken straight from the upstream docs, IIRC. I will leave it as-is for now, based on the response from @dankenigsberg - but let me know if you can clarify further.

@myakove Are there any other concerns from a QE perspective? I'm not sure how things get marked "Verified" in JIRA since the next status is "Done." I'm eager to merge this PR since it needs to go in the 1.4 docs. :) Thanks for your help.

Reply:
@myakove Just saw that you created a sub-task, so ignore my question about how things get marked "Verified." I assume that when you've finished testing this story, you will mark that sub-task "Done" which will indicate to me that I can merge the PR. Let me know if that's incorrect. Thanks!